libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -93,6 +93,8 @@ singleton AP_GPS method get_antenna_offset Vector3f uint8_t 0 ud->num_sensors()
 singleton AP_GPS method first_unconfigured_gps boolean uint8_t'Null
 
 include AP_Math/AP_Math.h
+-- Rotation for RangeFinder methods
+include AP_Math/rotations.h
```

Comment:
We already get rotations.h in AP_Math.h so I think this extra define is unnecessary.